src/app/campaign/flight/flight-form-control-container.component.ts

```diff
@@ -133,11 +133,16 @@ export class FlightFormControlContainerComponent implements OnInit, OnDestroy {
 
   validateEndAt(endAt: AbstractControl): { [key: string]: any } | null {
     if (
+      endAt.value &&
       this.flightActualsDateBoundaries &&
       this.flightActualsDateBoundaries.endAt &&
```

Comment:
The end date validation was no change because turns out I didn't do it correctly in the first place. Dates are complicated. 🤷

Reply:
Pre-fixed-forever!